app/presenters/eastern_time_presenter.rb

```diff
@@ -0,0 +1,21 @@
+class EasternTimePresenter
+  def initialize(timestamp)
+    @timestamp = timestamp
+  end
+
+  def to_s
+    I18n.t(
+      'event_types.timestamp',
+      date: eastern_timestamp.strftime('%B %e, %Y'),
```

Comment:
COOL! love that use of `I18n.l`

done here: f8f02129

note that I went without the `%z` since that output an integer and @andrewhughey was pretty clear in the GH issue that he wanted "Eastern" for clarity.